src/olympia/addons/models.py

```diff
@@ -669,6 +669,33 @@ def allow_resubmission(self):
     def disable_all_files(self):
         File.objects.filter(version__addon=self).update(status=amo.STATUS_DISABLED)
 
+    def set_needs_human_review_on_latest_versions(self, *, due_date=None, reason):
```

Comment:
I know it's valid python... but can we have fields with defaults after fields with no defaults?